client/src/features/signup/components/SignupForm/SignupForm.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Stepper, StepLabel, Step, Button, Box, CircularProgress } from '@mui/material';
+import { Formik, Form } from 'formik';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+
+import { initialValues, validationSchema } from '../../data';
+import { UserInfoForm, LoginInfoForm, SignupSuccess } from '..';
+
+const steps = ['User Info', 'Login Info'];
+
+const SignupForm = () => {
+  const [activeStep, setActiveStep] = useState(0);
+  const [loading, setLoading] = useState(false);
+  const currentValidationSchema = validationSchema[activeStep];
+  const lastStep = steps.length - 1;
+
+  const nextStep = () => {
+    setActiveStep((prev) => prev + 1);
+  };
+
+  const previousStep = () => {
+    setActiveStep((prev) => prev - 1);
+  };
+
+  const formSubmitHandler = (values, { setSubmitting }) => {
+    setLoading(true);
+
+    setTimeout(() => {
+      setLoading(false);
+      nextStep();
+      setSubmitting(false);
+    }, 2000);
+
+    // console.log(values);
```

Comment:
This console log can be removed

Reply:
its for future reference, in order to know what kind of object we are sending to the API and adjust it accordingly